src/main/resources/templates/polls/results.twig

```diff
@@ -18,18 +18,66 @@
 {% endblock %}
 
 {% block scripts %}
-	<script src="{{ asset('node_modules/d3/d3.min.js') }}"></script>
-	<script src="{{ asset('node_modules/d3pie/d3pie/d3pie.min.js') }}"></script>
+	<style>
+		.arc text {
+			font: 10px sans-serif;
+			text-anchor: middle;
+		}
+
+		.arc path {
+			stroke: #fff;
+		}
+	</style>
+	<script src="{{ asset('node_modules/d3/build/d3.min.js') }}"></script>
 	<script>
 		var options = []; // todo: do this with jsonify stuff
 		{% for option, nVotes in poll.options -%}
 		options.push({label: "{{option}}", value: {{ nVotes }}});
```

Comment:
should be `window.options` so that you can use `import {d3} from 'd3'` and move it into its own js file or part of js/results